libraries/AP_Notify/AP_Notify.cpp

```diff
@@ -115,6 +116,7 @@ void AP_Notify::init(bool enable_external_leds)
         _devices[1] = new ToshibaLED_I2C();
         _devices[2] = new ToneAlarm_PX4();
         _devices[3] = new Display();
+        _devices[4] = new mRoLED_I2C();
```

Comment:
I just worked on Pixhawk2.1 but I made a prototype module broken unfortunately. So that I'd like to ask somebody who has another LED module to continue testing via Randy-san. I faced hardware signal issue (LED color lit is not intended sometimes), probably due to the long jumper cabling.